SparkBuilderGeneratorPlugin/src/com/helospark/spark/builder/handlers/codegenerator/component/fragment/constructor/SuperSetterMethodAdderFragment.java

```diff
@@ -0,0 +1,39 @@
+package com.helospark.spark.builder.handlers.codegenerator.component.fragment.constructor;
+
+import java.util.List;
+
+import org.eclipse.jdt.core.dom.AST;
+import org.eclipse.jdt.core.dom.Block;
+import org.eclipse.jdt.core.dom.FieldAccess;
+import org.eclipse.jdt.core.dom.SuperMethodInvocation;
+
+import com.helospark.spark.builder.handlers.codegenerator.domain.SuperSetterBasedBuilderField;
+
+/**
+ * Adds setter calls for superclass fields, like:
+ * <pre>
+ * super.setField(builder.field);
+ * super.setOtherField(builder.otherField);
+ * </pre>
+ * @author helospark
+ */
+public class SuperSetterMethodAdderFragment {
```

Comment:
SuperClassSetterMethodAdderFragment?

Reply:
Fair enough, but maybe rather SuperFieldSetterMethodAdderFragment.
With `SuperClassSetterMethodAdderFragment`, I would associate adding setter methods to superclass.